### PR TITLE
Add SINGA Kohaku

### PR DESCRIPTION
**Pull request by itsme-zeix**
## Description

Add VIA support for SINGA Kohaku

## QMK Pull Request

https://github.com/qmk/qmk_firmware/pull/24309

## VIA Keymap Pull Request

https://github.com/the-via/qmk_userspace_via/pull/19/files

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] VIA keymap is **MERGED** in VIA userspace master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have formatted the JSON file to have consistent formatting with the rest of the repository.
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by Cipulot** [comment]
@itsme-zeix keymap in the VIA rep was approved, if this PR is ok to be reviewed for merge flag it and I'll get to it asap. Asking since I don't want to review and possibly merge something that may not be 100% final and up to date, flagging it will tell me that it's ready. 
Cheers

**Comment by itsme-zeix** [comment]
@Cipulot Done! Thank you so much for the quick review! Really appreciate the work you've done taking over some of the VIA maintenance.

File: v3/zeix/singa/kohaku/kohaku.json [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+            },
+            "8,5\n\n\n3,3",
+            {
+              "w": 1.25
+            },
+            "9,5\n\n\n3,3"
+          ]
+        ]
+  }
+}
```
Comment (Cipulot) [inline v3/zeix/singa/kohaku/kohaku.json:306]:
```suggestion
{
  "name": "SINGA Kohaku",
  "vendorId": "0x4C27",
  "productId": "0x8888",
  "matrix": {
    "rows": 10,
    "cols": 8
  },
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgblight"
  ],
  "layouts": {
    "labels": [
      "Split Backspace",
      "ISO Enter",
      "Split Left Shift",
      [
        "Bottom Row",
        "7u",
        "6.25u",
        "Split Space (2.75u Left)",
        "Split Space (2.25u Left)"
      ]
    ],
    "keymap": [
      [
        {
          "x": 2.75,
          "c": "#777777"
        },
        "0,0",
        {
          "c": "#cccccc"
        },
        "1,0",
        "0,1",
        "1,1",
        "0,2",
        "1,2",
        "0,3",
        "1,3",
        "0,4",
        "1,4",
        "0,5",
        "1,5",
        "0,6",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "0,7\n\n\n0,0",
        {
          "c": "#cccccc"
        },
        "1,7",
        {
          "x": 0.75
        },
        "1,6\n\n\n0,1",
        "0,7\n\n\n0,1"
      ],
      [
        {
          "x": 2.75,
          "c": "#aaaaaa",
          "w": 1.5
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "3,0",
        "2,1",
        "3,1",
        "2,2",
        "3,2",
        "2,3",
        "3,3",
        "2,4",
        "3,4",
        "2,5",
        "3,5",
        "2,6",
        {
          "w": 1.5
        },
        "3,6\n\n\n1,0",
        "3,7",
        {
          "x": 1.5,
          "c": "#777777",
          "w": 1.25,
          "h": 2,
          "w2": 1.5,
          "h2": 1,
          "x2": -0.25
        },
        "5,6\n\n\n1,1"
      ],
      [
        {
          "x": 2.75,
          "c": "#aaaaaa",
          "w": 1.75
        },
        "4,0",
        {
          "c": "#cccccc"
        },
        "5,0",
        "4,1",
        "5,1",
        "4,2",
        "5,2",
        "4,3",
        "5,3",
        "4,4",
        "5,4",
        "4,5",
        "5,5",
        {
          "c": "#777777",
          "w": 2.25
        },
        "5,6\n\n\n1,0",
        {
          "c": "#cccccc"
        },
        "5,7",
        {
          "x": 0.5
        },
        "4,6\n\n\n1,1"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "6,0\n\n\n2,1",
        "7,0\n\n\n2,1",
        {
          "x": 0.5,
          "w": 2.25
        },
        "6,0\n\n\n2,0",
        {
          "c": "#cccccc"
        },
        "6,1",
        "7,1",
        "6,2",
        "7,2",
        "6,3",
        "7,3",
        "6,4",
        "7,4",
        "6,5",
        "7,5",
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "6,6",
        {
          "c": "#cccccc"
        },
        "7,6",
        "7,7"
      ],
      [
        {
          "x": 2.75,
          "c": "#aaaaaa",
          "w": 1.5
        },
        "8,0\n\n\n3,0",
        "9,0\n\n\n3,0",
        {
          "w": 1.5
        },
        "8,1\n\n\n3,0",
        {
          "c": "#cccccc",
          "w": 7
        },
        "8,3\n\n\n3,0",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "9,5\n\n\n3,0",
        {
          "x": 0.5,
          "c": "#cccccc"
        },
        "8,6",
        "9,6",
        "9,7"
      ],
      [
        {
          "y": 0.5,
          "x": 2.75,
          "c": "#aaaaaa",
          "w": 1.25
        },
        "8,0\n\n\n3,1",
        {
          "w": 1.25
        },
        "9,0\n\n\n3,1",
        {
          "w": 1.25
        },
        "8,1\n\n\n3,1",
        {
          "c": "#cccccc",
          "w": 6.25
        },
        "8,3\n\n\n3,1",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "8,5\n\n\n3,1",
        {
          "w": 1.25
        },
        "9,5\n\n\n3,1"
      ],
      [
        {
          "x": 2.75,
          "w": 1.25
        },
        "8,0\n\n\n3,2",
        {
          "w": 1.25
        },
        "9,0\n\n\n3,2",
        {
          "w": 1.25
        },
        "8,1\n\n\n3,2",
        {
          "c": "#cccccc",
          "w": 2.75
        },
        "9,2\n\n\n3,2",
        {
          "w": 1.25
        },
        "8,3\n\n\n3,2",
        {
          "w": 2.25
        },
        "9,4\n\n\n3,2",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "8,5\n\n\n3,2",
        {
          "w": 1.25
        },
        "9,5\n\n\n3,2"
      ],
      [
        {
          "x": 2.75,
          "w": 1.25
        },
        "8,0\n\n\n3,3",
        {
          "w": 1.25
        },
        "9,0\n\n\n3,3",
        {
          "w": 1.25
        },
        "8,1\n\n\n3,3",
        {
          "c": "#cccccc",
          "w": 2.25
        },
        "9,2\n\n\n3,3",
        {
          "w": 1.25
        },
        "8,3\n\n\n3,3",
        {
          "w": 2.75
        },
        "9,4\n\n\n3,3",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "8,5\n\n\n3,3",
        {
          "w": 1.25
        },
        "9,5\n\n\n3,3"
      ]
    ]
  }
}

```

**Review by Cipulot — CHANGES_REQUESTED** [review]

File: v3/zeix/singa/kohaku/kohaku.json (hunk 1 above)
Comment (Cipulot) [reply v3/zeix/singa/kohaku/kohaku.json:306]: Formatting and adding lighting keycodes since lighting is enabled.

**Review by Cipulot — APPROVED** [review]

**Pull request merged**